scripts/kickstart_generator.rb

```diff
@@ -23,17 +23,17 @@ def initialize(build_base, targets, puddle, manageiq_checkout, appliance_checkou
       @ssui_checkout      = ssui_checkout
     end
 
-    def run(task = :all)
+    def run
+      FileUtils.mkdir_p(@ks_gen_base)
+
       targets.each do |target|
         @target = target # used during ERB evaluation
 
         result = evaluate_erb
 
-        FileUtils.mkdir_p(@ks_gen_base)
-        File.write(@ks_gen_base.join("base.ks"), result)
-
-        write_config(result) if [:all, :config].include?(task)
-        write_json(result)   if [:all, :json].include?(task)
+        file = @ks_gen_base.join("base-#{@target}.ks")
+        $log.info("Writing kickstart file: #{file}") if $log
+        File.write(file, result)
```

Comment:
Ok, thanks @geertj.  I like the "new" way, having two formats was silly.  I don't know that we need both formats still though or if we can assume brew/koji and our upstream build machine vms have the required imagefactory versions to take advantage of this.